src/test/data/XmlBookShelfStorageTest/invalidAndValidBookBookShelf.xml

```diff
@@ -4,6 +4,7 @@
     <books>
         <title>Artemis</title>
         <description>This is Artemis.</description>
+        <rate>Rate</rate>>
```

Comment:
Remove the extra `>` at the end of the line. Do the same for the other `.xml` files you modified.